nncf/onnx/graph/nncf_graph_builder.py

```diff
@@ -64,6 +67,7 @@ def __init__(
         self.weight_attrs = weight_attrs if weight_attrs is not None else {}
         self.bias_attrs = bias_attrs if bias_attrs is not None else {}
         self.node_attrs = node_attrs if node_attrs is not None else {}
+        self._layer_attributes = layer_attributes
```

Comment:
The `ONNXLayerAttributes` is already a `BaseLayerAttributes`. This is highly confusing to anyone reading this and is a consequence of a poorly designed `LayerAttributes` as a whole.

Reply:
Outdated